test/fixtures/schedules.yml

```
@@ -11,3 +11,10 @@ ls_on_atahualpa:
   end: <%= 1.year.from_now.to_s :db %>
   interval: 1
   frequency: weeks
+
+boom_on_atahualpa:
+  name: 'Chica pumba chica pum'
```

Te dejo el "Are you reeeeaaaadddddyyy?", pero en este vamos con algo aburrido como:
```suggestion
  name: 'Scheduled boom'
```